python/dgl/ops/segment.py

```diff
@@ -50,7 +50,7 @@ def segment_reduce(seglen, value, reducer='sum'):
     if len(u) != len(v):
         raise DGLError("Invalid seglen array:", seglen,
                        ". Its summation must be equal to value.shape[0].")
-    g = convert.bipartite((u, v))
+    g = convert.heterograph({('_U', '_E', '_V'): (u, v)})
```

Comment:
Can we omit ('_U', '_E', '_V')? it is a little bit ugly.

Reply:
Previously this is also what happed with `convert.bipartite`...

Reply:
I'm fine with this actually.